runner.yml.dist

```diff
@@ -23,8 +23,10 @@ drupal:
     - "./vendor/bin/drush config-set system.theme admin seven -y"
     - "./vendor/bin/drush config-set node.settings use_admin_theme 1 -y"
     # Enable the modules.
+    - "./vendor/bin/drush en oe_media -y"
```

Comment:
Since oe_content_news depends on this, we don't need to enable manually no?

Reply:
In this case, yes we don't need.